app/frontend/src/utils/constants.js

```diff
@@ -4,7 +4,8 @@ export const ApiRoutes = Object.freeze({
   DOCS: '/docs',
   EMAIL: '/email',
   HEALTH: '/checks/status',
-  KCCONFIG: '/keyCloak/configForm',
+  KCCLIENTS: '/keycloak/serviceClients',
+  KCCONFIG: '/keycloak/configForm',
```

Comment:
Probably just a nitpick, but we should stay consistent - either we change these names to KC_CLIENTS and KC_CONFIG, or change WEBADE_CONFIGFORM to WEBADECONFIGFORM. I personally lean towards the underscore, but consistent naming either way is more important.

Reply:
Changed to _